sql/updates/world/3.3.5/XXXX_XX_XX_XX_world.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_item_party_time';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (8067, 'spell_item_party_time');
```

Comment:
codestyle/assurance: add comment on first line, in case previously loaded SQL file does not have NewLine at EOF
```suggestion
-- 
DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_item_party_time';
INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (8067, 'spell_item_party_time');
```